containers/daap-python-base/src/var/task/versioning.py

```diff
@@ -8,10 +8,15 @@
 
 import boto3
 from data_platform_logging import DataPlatformLogger, s3_security_opts
-from data_platform_paths import DataProductConfig
+from data_platform_paths import (
+    DataProductConfig,
+    DataProductElement,
+    delete_all_element_version_data_files,
```

Comment:
Don't want this import anymore as it's in versioning now

Reply:
Agreed now moved.